src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+  
+  </Side>
+  </SideBySide>
+
+# Errors inbox [#errors-inbox]
+<SideBySide>
+  <Side>
+
+  Errors inbox is an error-tracking solution. It's designed to put you in control. It helps you find, focus on, and fix errors across your application stack. Errors inbox is embedded in the <InlinePopover type="apm" /> UI so you can resolve errors faster.
+
+  Learn more about errors inbox [here](/docs/apm/errors-inbox/errors-inbox-ui).
+
+  </Side>
+  <Side>
+
+  <img
+  title="New Relic Errors inbox"
+  alt="New Relic Errors inbox"
+  src={NRErrosInboxMain}
+  />
+
+  </Side>
+  </SideBySide>
+
+# IAST [#iast]
+<SideBySide>
+  <Side>
+
+  <img
+  title="New Relic IAST"
+  alt="New Relic IAST"
+  src={NRIastMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic Interactive Application Security Testing (IAST) can help you stop cyberattacks and breaches on your applications. It does this by probing your running code for exploitable flaws.
+
+  Learn more about IAST [here](/docs/iast/introduction).
+
+  </Side>
+  </SideBySide>
+
+# Infrastructure [#infrastructure]
+<SideBySide>
+  <Side>
+
+  The <InlinePopover type="infrastructure" /> capability lets you monitor the health and performance of your service. It covers containers and cloud infrastructure. It includes tracking CPU, memory, network traffic, and disk use. It also includes over 400 on-host integrations for monitoring third-party apps. 
+
+  Learn more about infrastructure [here](/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring/).
+
+  </Side>
+  <Side>
+
+  <img
+    title="New Relic infrastructure"
+    alt="New Relic infrastructure"
+    src={NRInfraMain}
+  />
+  </Side>
+  </SideBySide>
+
+# Kubernetes and Pixie [#k8s-pixie]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic Kubernetes"
+    alt="New Relic Kubernetes"
+    src={NRK8sMain}
+  />
+
+  </Side>
+  <Side>
+
+  The New Relic Kubernetes integration gives you full observability. The data it provides lets you monitor the health of your whole Kubernetes cluster and the performance of your environment. You can check individual pods or drill into specific services and applications. The agent collects telemetry data from your cluster. It uses several New Relic integrations. These include Kubernetes events, the Prometheus Agent, and the New Relic Logs Kubernetes plugin.
+
+  Learn more about Kubernetes [here](l/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration).
+
+  </Side>
+  </SideBySide>
+
+# Live archives [#live-archives]
+<SideBySide>
+  <Side>
+
+  Live archives store historical logs with other logs and telemetry data in the New Relic Database. This allows instant access to historical logs and eliminates the need to rehydrate, reload, re-index, or move data to many places. 
```

Comment:
Is rehydrate the right term here? I'm not sure what that means in this context, I think you can cut it out.

Reply:
haha, I copied from here: https://newrelic.com/blog/nerdlog/live-archives